modules/util/source-formatter/src/main/resources/sourcechecks.xml

```diff
@@ -1592,6 +1592,10 @@
 			<category name="Upgrade" />
 			<description name="Replaces the references of the method `HtmlUtil.extractText(` with the method `extractText(` of `HtmlParser` class" />
 		</check>
+		<check name="UpgradeJavaServiceReferenceAnnotationCheck">
+			<category name="Upgrade" />
+			<description name="Run code migration of Annotation '@ServiceReference' to '@Reference'" />
```

Comment:
@TamyrisTorres, please rephrase to: 

**Run code migration to replace @ServiceReference by @ Reference**